docker-compose.yml

```diff
@@ -7,10 +6,13 @@ services:
     ports:
       - 3000:3000
     volumes:
-      - ./testCourse:/course
       - /var/run/docker.sock:/var/run/docker.sock
       - ${HOME}/pl_ag_jobs:/jobs
       - .:/PrairieLearn
+      # To test local changes to workspaces or external graders
+      # or override other defaults from ConfigSchema,
+      # enable the following line and make your edits.
+      # - config.dev.json:/PrairieLearn/config.json
```

Comment:
Is there a reason you don't use `config.json`? Then you wouldn't need to map it in at a different location.

Reply:
A `config.json` would override defaults. This allows you to be more explicit about accepting or not accepting defaults.

Reply:
This change was since removed. If there's anything still relevant here, let's capture it in a comment on https://github.com/PrairieLearn/PrairieLearn/issues/11934.